### PR TITLE
Add links to resources containing background info

### PR DESCRIPTION
**Pull request by eojthebrave**
At  the recent collaborators summit breakout about website re-design there were some useful links shared that contain a bunch of background information about the project. For someone who is just trying to figure out how I can help, and what the current state of things is, these were useful. Unfortunately I wouldn't have known about them if I hadn't been in the room when they were mentioned.

I think it might also be worthwhile to add links to one or two issues or areas of discussion that the group is currently focused on. This would help direct attention of anyone new to the project, or just curious about where things are at. (Heh, including me who is still trying to figure that out 😄). I don't know what those are though, so would be great if someone more familiar with the project could either add them, or let me know so I can add them.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,10 @@ own [Contributing Guidelines][].
 [Meeting notes][] from past meetings are maintained for later reference.
 [Query issues for "Meeting"][] to find future meetings.
 
+## Summary / Current Status
+
+Our current focus is on UI design and content creation. You can get some background information by reading these two articles: [Redesigning Nodejs.org Part 1](https://medium.com/the-node-js-collection/redesigning-nodejs-part-1-fac08a0e015a) and [Redesigning Nodejs.org Part 2](https://medium.com/@ammiller/redesigning-nodejs-org-584212bb25e1). And browser the current working wirefames here https://projects.invisionapp.com/share/SJG12D028NX#/screens/281335816.
```
Comment (keywordnew) [inline README.md]: Thanks for position it near the top for easy visibility!

Two small nits: typo, and linking with markdown. I'm using the new suggestion feature. It's awesome.

```suggestion
Our current focus is on UI design and content creation. You can get some background information by reading these two articles: [Redesigning Nodejs.org Part 1](https://medium.com/the-node-js-collection/redesigning-nodejs-part-1-fac08a0e015a) and [Redesigning Nodejs.org Part 2](https://medium.com/@ammiller/redesigning-nodejs-org-584212bb25e1). And browse the current working wirefames [here](https://projects.invisionapp.com/share/SJG12D028NX#/screens/281335816).
```

**Review by keywordnew — APPROVED** [review]
Firstly...
screen shot 2018-10-16 at 11 41 30 pm
Woop woop! 🙌🏽

LGTM, with 2 nits.

**Comment by eojthebrave** [comment]
Hopefully the first of many. 😁 Thanks for the review.

**Review by sagirk — APPROVED** [review]
LGTM! 🎉

Thanks for the contribution. ❤️

**Pull request merged**